src/components/Navbar/Navbar.jsx

```diff
@@ -1,43 +1,67 @@
 import React from "react";
 import "./Navbar.css";
 import { useNavigate } from "react-router-dom";
+import { useNotes } from "../../context/notes-context";
```

Comment:
Can add barrel imports to make the imports look a bit cleaner. Check out this article - https://dev.to/justmyrealname/organize-react-components-better-with-barrel-exports-4fno

Reply:
Thanks, I will check this out